server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -328,10 +328,12 @@ private void scheduleJobs(List<JobEntry> pendingJobs) {
         }
         try {
             for (JobEntry jobentry : pendingJobs) {
+                //Trigger cron jobs with higher priority than async ( default 5 )
                 Trigger trigger = newTrigger()
                     .withIdentity(jobentry.getJobName(), CRON_GROUP)
                     .withSchedule(cronSchedule(jobentry.getSchedule())
-                        .withMisfireHandlingInstructionDoNothing())
+                    .withMisfireHandlingInstructionDoNothing())
+                    .withPriority(3)
```

Comment:
I think this is backwards if we want the cron jobs to have priority.  The docs [state](http://www.quartz-scheduler.org/api/2.2.1/org/quartz/TriggerBuilder.html):

```
Set the Trigger's priority.  When more than one Trigger have the same fire time
the scheduler will fire the one with the highest priority first.
```
So the higher the number the higher the priority.

Looking through the [source](https://github.com/quartz-scheduler/quartz/blob/a9c39277fa1ca983d1bc384e81909f37f8fae221/quartz-core/src/main/java/org/quartz/impl/jdbcjobstore/StdJDBCConstants.java#L509) seems to confirm this since they have

```
+ "ORDER BY " + COL_NEXT_FIRE_TIME + " ASC, " + COL_PRIORITY + " DESC";
```

where higher a higher priority number would precede a lower one.

Reply:
@awood my bad. I misread. fixed.